srtcore/api.cpp

```diff
@@ -1507,17 +1526,19 @@ void CUDTUnited::removeSocket(const SRTSOCKET u)
    if (i == m_ClosedSockets.end())
       return;
 
+   CUDTSocket* s = i->second;
+
```

Comment:
can it be const?

Reply:
Depends what exactly. The pointer itself, sure. The designated object (reference) not.